packages/host/app/components/operator-mode/code-submode/playground/playground-panel.gts

```diff
@@ -191,11 +263,13 @@ export default class PlaygroundPanel extends Component<Signature> {
   }
 
   private get canEditCard() {
-    return Boolean(
-      this.format !== 'edit' &&
-        this.card?.id &&
-        this.realm.canWrite(this.card.id),
-    );
+    // in error case, we may not have `this.card?.id`, but we can get the id from `this.cardResource?.id`
+    let cardId = this.cardResource?.id;
```

Comment:
ditto on the remote vs local ID subtlety that was introduced on this line as above